jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/DiagnosticUtils.java

```diff
@@ -472,6 +919,32 @@ private static DiagnosticProperties doCheckpointJVM(String diagnosticCommand) {
 			+ "          agentLibrary: the absolute path of the agent%n"
 			+ "          agent option: (Optional) the agent option string%n";
 
+/*[IF JFR_SUPPORT]*/
+/*[IF JAVA_SPEC_VERSION == 11]*/
+	private static final String DIAGNOSTICS_JFR_START_HELP = "Starts a new Recording%n%n"
+			+ JFR_FORMAT_PREFIX + DIAGNOSTICS_JFR_START + JFR_START_OPTION_HELP;
+
+	private static final String DIAGNOSTICS_JFR_STOP_HELP = "Stops a JFR recording%n%n"
+			+ JFR_FORMAT_PREFIX + FORMAT_PREFIX + DIAGNOSTICS_JFR_STOP + JFR_STOP_OPTION_HELP;
+
+	private static final String DIAGNOSTICS_JFR_DUMP_HELP = "Copies contents of a JFR recording to file. Either the name or the recording id must be specified.%n%n"
+			+ JFR_FORMAT_PREFIX + DIAGNOSTICS_JFR_DUMP + JFR_DUMP_OPTION_HELP;
+/*[ENDIF] JAVA_SPEC_VERSION == 11 */
+/*[ENDIF] JFR_SUPPORT */
+
+/*[IF JFR_SUPPORT]*/
+/*[IF JAVA_SPEC_VERSION == 17]*/
```

Comment:
Does this apply to JDK 21+?

Reply:
`JFR_SUPPORT` is enabled by default at platform `xa64`.
The changes must only be consistent for JDK 11/17, and not exposed to JDK 21+.

Reply:
We are currently validating JFR framework for 17+ as well. I will update this PR once done.